datapunt_api/static/.release-it.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "https://unpkg.com/release-it/schema/release-it.json",
+    "git": {
+      "requireCleanWorkingDir": true,
+      "tagName": "v${version}",
+      "commitMessage": "Release v${version}",
+      "tagAnnotation": "Release v${version}",
+      "push": true
+    },
+    "github": {
+      "release": true,
+      "pr": {
+        "title": "chore(release): ${version}",
+        "body": "This PR includes the version bump and changelog updates for release ${version}.",
+        "base": "main"
+      }
+    },
+    "npm": {
+      "publish": false
+    },
+    "plugins": {
+      "@release-it/conventional-changelog": {
+        "preset": "angular"
```

Comment:
I think we prefer "conventionalcommits" as a preset. that's how we set it up with ee-docs